svf-llvm/lib/LLVMUtil.cpp

```diff
@@ -511,7 +510,7 @@ void LLVMUtil::removeFunAnnotations(Set<Function*>& removedFuncList)
     glob->setName("llvm.global.annotations.old");
     GlobalVariable *GV = new GlobalVariable(newCA->getType(), glob->isConstant(), glob->getLinkage(), newCA, "llvm.global.annotations");
     GV->setSection(glob->getSection());
-    module->getGlobalList().push_back(GV);
+    module->insertGlobalVariable(GV);
```

Comment:
Thanks for the contribution. Could you add a macro for this?

#if (LLVM_VERSION_MAJOR < 17)
    module->getGlobalList().push_back(GV);
#elif (LLVM_VERSION_MAJOR >= 17)
    module->insertGlobalVariable(GV);
#else
    assert(false && "llvm version not supported!");
#endif

Reply:
I added the macro.